src/classes/ALLO_PaymentSync_SVC.cls

```diff
@@ -218,6 +233,114 @@ public class ALLO_PaymentSync_SVC {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Given a set of Payment Ids, and a set of Opportunity Ids, this method
+    * returns the Payments associated with them, as well as any associated Allocations.
+    * @param Set<Id> paymentIds A set of Payment Ids
+    * @param Set<Id> opportunityIds A set of Opportunity Ids
+    * @return Map<Id, npe01__OppPayment__c> map of Payments with Payment Allocations by Payment Id
+    ********************************************************************************************************/
+    @testVisible Map<Id, npe01__OppPayment__c> retrievePaymentsWithAllocations(Set<Id> paymentIds, Set<Id> opportunityIds) {
```

Comment:
Although the default method modifier in Apex is private. Should we indicate here to make it more clear?